### ISSUE TITLE
Telegram notifications don't include metrics (again)

### ISSUE THREAD

**Issue by maxkoshevoi** [sadviq99/db1000n-setup#4]
Seems like they've updated the template again 😅

**Comment by sadviq99** [comment]
🥲 
working on it

**Comment by sadviq99** [comment]
Actually, they moved completely from the table output to a new format. The worse part is that there is no `Total` value anymore, it means I can either:
- specify metrics per target, that will increase a size of TG notification, making it less readable 
- calculate summary by myself, but it will lead to a lot of values conversion which potentially may result to incorrect data

I created issue https://github.com/Arriven/db1000n/issues/516 in their repo with a question on how to get the total value, will see what they reply. Hopefully, there is a way to get the `Total` values natively, otherwise, I'll work on this by myself.

**Comment by vkislichenko** [comment]
@sadviq99 there is [prometheus integration ](https://github.com/Arriven/db1000n/pull/219) (though I was not able to access it)
not sure what format/data it should respond with, but it seems like "the way" to gather/get data

**Comment by sadviq99** [comment]
> @sadviq99 there is [prometheus integration ](https://github.com/Arriven/db1000n/pull/219) (though I was not able to access it) not sure what format/data it should respond with, but it seems like "the way" to gather/get data

As I understand, you need to provide a Prometheus URL to db1000n so it will send the metrics to that endpoint. Prometheus can be integrated with Grafana/Kibana to nicely visualise the data, but it means someone needs to host & maintain the Prometheus server. It's a bit too much for our use case, all we need is db1000n to provide a singe log line with a total traffic calculation, as it was before. Lets wait for their answer for a bit, if nothing concrete - I'll re-invent the wheel. Or everyone is very welcomed to submit a PR 😊

**Comment by sadviq99** [comment]
@maxkoshevoi @vkislichenko I addressed the issue in PR #6, please let me know if you have any problems with the update.

**Comment by maxkoshevoi** [comment]
Thank you! Could you also add a %? How it was in the initial db1000n metrics?

**Comment by sadviq99** [comment]
@maxkoshevoi unfortunately, no. They removed the response rate % as well.
The most useful metric, as to me.

**Comment by maxkoshevoi** [comment]
Yeah, but maybe it could be calculated? Like `sent / received`, or it's more complicated than that?

**Comment by sadviq99** [comment]
Something like this, but `sent` and `received` should also be calculated globally, for all of the targets. Doable, but need some testing. I will look into that a bit later, or you can submit a PR.

**Comment by tactusa** [comment]
@sadviq99 Hey, seems like notifications are failing again 😅

**Comment by sadviq99** [comment]
Yeah... I can't work on these nearest days, but I'll get this fixed as soon as I can. 
Hopefully, db1000n team will stabilize the output format till that time 🤞

**Comment by maxkoshevoi** [comment]
Notifications are fixed now (without applying fixes from #6). Have they returned the overall stats?

**Comment by tactusa** [comment]
@sadviq99 Hey, i've seen latest commit and tried this update, but seems like it's not working still. Has it fixed the issue and is it working now for you?
Thanks

**Comment by MaxBogdan** [comment]
Looks like there is an error in script to call docker compose: `docker-compose` used instead of `docker compose`.
Once I changed it (removed "-" dash) it works

@sadviq99 FYI

**Comment by sadviq99** [comment]
> Notifications are fixed now (without applying fixes from #6). Have they returned the overall stats?

Seems like they, but this may be changed again anytime.
